backend/src/email/middlewares/email-transactional.middleware.ts

```diff
@@ -128,6 +149,40 @@ export const InitEmailTransactionalMiddleware = (
       tag,
       // not sure why unknown is needed to silence TS (yet other parts of the code base can just use `as Model` directly hmm)
     } as unknown as EmailMessageTransactional)
+
+    // save cc and bcc
+    let transactionalCcEmails: EmailMessageTransactionalCc[] = []
+
+    transactionalCcEmails = transactionalCcEmails.concat(
+      cc
+        ? cc.map(
+            (c) =>
+              ({
+                emailMessageTransactionalId: emailMessageTransactional.id,
+                email: c,
+                ccType: CcType.Cc,
+              } as EmailMessageTransactionalCc)
+          )
+        : []
+    )
+
+    transactionalCcEmails = transactionalCcEmails.concat(
+      bcc
+        ? bcc.map(
+            (c) =>
+              ({
+                emailMessageTransactionalId: emailMessageTransactional.id,
+                email: c,
+                ccType: CcType.Bcc,
+              } as EmailMessageTransactionalCc)
+          )
+        : []
+    )
+
+    if (transactionalCcEmails && transactionalCcEmails.length > 0) {
+      await EmailMessageTransactionalCc.bulkCreate(transactionalCcEmails)
```

Comment:
Assign `emailMessageTractional.emailMessageTransactionalCc` to this returned value then u won't need to use the extra arg in the convert fn